From 9c4e120a7a87db34d22a50883a5a525170b480d7 Mon Sep 17 00:00:00 2001
From: Robert Yang <liezhi.yang@windriver.com>
Date: Tue, 6 Jan 2015 23:10:51 -0800
Subject: [PATCH] libguile/Makefile.am: install-data-hook: depends on
 install-libLTLIBRARIES

It may install such a file:
/usr/lib64/libguile-2.0*-gdb.scm

This is because when there is no file in the directory:
for f in libguile-2.0*; do
    [snip]
done

The f would be libguile-2.0* itself.

Upstream-Status: Pending

Signed-off-by: Robert Yang <liezhi.yang@windriver.com>
---
 libguile/Makefile.am |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libguile/Makefile.am b/libguile/Makefile.am
index 281faac..fe0a3ba 100644
--- a/libguile/Makefile.am
+++ b/libguile/Makefile.am
@@ -449,7 +449,7 @@ EXTRA_libguile_@GUILE_EFFECTIVE_VERSION@_la_SOURCES = _scm.h		\
 install-exec-hook:
 	rm -f $(DESTDIR)$(bindir)/guile-snarf.awk
 
-install-data-hook: libguile-2.0-gdb.scm
+install-data-hook: libguile-2.0-gdb.scm install-libLTLIBRARIES
 	@$(MKDIR_P) $(DESTDIR)$(libdir)
 ## We want to install libguile-2.0-gdb.scm as SOMETHING-gdb.scm.
 ## SOMETHING is the full name of the final library.  We want to ignore
-- 
1.7.9.5

